util/save-file-utils.js

```diff
@@ -160,6 +160,9 @@ module.exports = (() => {
 		    return pad + str;
 		},
 		typeSet: {
+		  '8utf8': ["string", 8, "utf-8"],
```

Comment:
The typeset argument of jBinary [require a fixed length parameter](https://github.com/jDataView/jBinary/wiki/Standard-types#strings) so I added several types of difference maximum length. However after some tests, I figured out that jBinary did not truncate the string to '0x00', we need to truncate the string manually. so maybe only one data type with a maximum length is sufficient.